tests/console/perl_bootloader.pm

```diff
@@ -50,7 +50,8 @@ sub run {
         trup_call 'run pbl --install';
         if (get_var('FLAVOR') =~ m/-encrypted/i) {
             # workaround bsc#1228126 poo#164021 poo#164156
-            script_run('cp /boot/efi/EFI/BOOT/sealed.tpm /boot/efi/EFI/sl');
+            script_run('cp /boot/efi/EFI/BOOT/sealed.tpm /boot/efi/EFI/sl') unless is_leap_micro('>6.0');
```

Comment:
Leap? Not `is_sle_micro`?

Reply:
I just used the opposite condition for added `if is_leap_micro('>6.0');` :sweat_smile: